portal-api/src/v1/controllers/banks.controller.js

```diff
@@ -1,9 +1,8 @@
+const { isVerifiedPayload, BANK } = require('@ukef/dtfs2-common');
```

Comment:
fwiw importing `BANK` from `@ukef/dtfs2-common` isn't very clear what it's going to be